src/components/Settings.vue

```diff
@@ -161,6 +161,14 @@
                           @click:append-outer="randomKey"
                         ></v-text-field>
                       </v-col>
+                      <v-col cols="12" sm="6">
+                        <v-switch
+                          hint="Metrics allows us to gain insight how `zwave-js` is used, which manufacturers and devices are most prevalent and where to best focus our efforts in order to improve `zwave-js` the most. We do not store any personal information. Details can be found under https://zwave-js.github.io/node-zwave-js/#/getting-started/telemetry.md#usage-statistics"
```

Comment:
Maybe we should do one iteration with @blhoward2 on how to formulate this best. Also affects zwave-js docs.

Reply:
👍🏼